Userland/Libraries/LibC/unistd.cpp

```diff
@@ -975,8 +975,49 @@ int unveil(char const* path, char const* permissions)
 // https://pubs.opengroup.org/onlinepubs/9699919799/functions/getpass.html
```

Comment:
this link is bogus. `getpass` was removed from POSIX.1 2001. A better link would be from the SUS, e.g. https://pubs.opengroup.org/onlinepubs/7908799/xsh/getpass.html